ui/v2.5/src/components/Shared/styles.scss

```diff
@@ -640,34 +640,31 @@ button.btn.favorite-button {
   display: inline-block;
 }
 
-.dropdown-menu:has(.scraper-filter-container) {
-  padding-top: 0;
-}
-
 .scraper-menu .dropdown-menu {
   min-width: 250px;
+  padding-top: 0;
 
   .dropdown-divider {
     border-top-color: $textfield-bg;
     margin: 0;
   }
 
   .scraper-filter-container {
-    background-color: $dark-gray5;
-    border-bottom: solid 1px rgba(16, 22, 26, 0.3);
+    background-color: $secondary;
+    border-bottom: solid 1px $textfield-bg;
     padding: 5px;
     position: sticky;
     top: 0;
     z-index: 1;
 
     .btn-group {
-      border: solid 1px rgba(16, 22, 26, 0.3);
+      border: solid 1px $textfield-bg;
       border-radius: 5px;
       width: 100%;
     }
 
     .clearable-text-field {
-      background-color: #26333d;
```

Comment:
Good catch. I picked up the color by using the color picker tool on one of the text fields on the page. At the time, it didn't cross my mind that the opacity applied to those fields would not be applied to the color code generated by the color picker.